docs/book/v6/tutorials/create-book-module.md

```diff
@@ -7,65 +7,102 @@ The below files structure is what we will have at the end of this tutorial and i
+                    ├──Repository/
+                    │   └──BookRepository.php
+                    └── ConfigProvider.php
+    
 ```
 
-* `src/Book/src/Collection/BookCollection.php` - a collection refers to a container for a group of related objects, typically used to manage sets of related entities fetched from a database
-* `src/Book/src/Entity/Book.php` - an entity refers to a PHP class that represents a persistent object or data structure
-* `src/Book/src/Handler/BookHandler.php` - handlers are middleware that can handle requests based on an action
-* `src/Book/src/Repository/BookRepository.php` - a repository is a class responsible for querying and retrieving entities from the database
-* `src/Book/src/Service/BookService.php` - is a class or component responsible for performing a specific task or providing functionality to other parts of the application
-* `src/Book/src/ConfigProvider.php` -  is a class that provides configuration for various aspects of the framework or application
-* `src/Book/src/RoutesDelegator.php` - a routes delegator is a delegator factory responsible for configuring routing middleware based on routing configuration provided by the application
-* `src/Book/src/InputFilter/BookInputFilter.php` - input filters and validators
-* `src/Book/src/InputFilter/Input/*` - input filters and validator configurations
+* `src/Book/src/Collection/BookCollection.php` – a collection refers to a container for a group of related objects, typically used to manage sets of related entities fetched from a database
+* `src/Book/src/ConfigProvider.php` – is a class that provides configuration for various aspects of the framework or application
+* `src/Book/src/Handler/GetBookCollectionHandler.php` – handler that reflects the GET action for the BookCollection class
+* `src/Book/src/Handler/GetBookResourceHandler.php` – handler that reflects the GET action for the Book entity
+* `src/Book/src/Handler/PostBookResourceHandler.php` – handler that reflects the POST action for the Book entity
+* `src/Book/src/InputFilter/CreateBookInputFilter.php` – input filters and validators
+* `src/Book/src/InputFilter/Input/*` – input filters and validator configurations
+* `src/Book/src/RoutesDelegator.php` – a routes delegator is a delegator factory responsible for configuring routing middleware based on routing configuration provided by the application
+* `src/Book/src/Service/BookService.php` – is a class or component responsible for performing a specific task or providing functionality to other parts of the application
+* `src/Core/src/Book/src/ConfigProvider.php` – is a class that provides configuration for Doctrine ORM
+* `src/Core/src/Book/src/Entity/Book.php` – an entity refers to a PHP class that represents a persistent object or data structure
+* `src/Core/src/Book/src/Repository/BookRepository.php` – a repository is a class responsible for querying and retrieving entities from the database
 
 ## Creating and configuring the module
```

Comment:
This section could be a bit confusing for devs.
You instruct them to create a `ConfigProvider` with classes which don't exist yet - better leave it for when all the classes have been created.
We should keep here only the instructions for creating the directory structure and move the creation of the two `ConfigProvider`s and the `RouteDelegator` to the end of the `File creation and contents` section.